.github/workflows/main.yaml

```diff
@@ -10,7 +13,7 @@ on:
 jobs:
   run-tests:
     runs-on: ubuntu-latest
-    container: ghcr.io/tiiuae/fog-ros-baseimage-builder:sha-7fca749
+    container: ros:iron
```

Comment:
```suggestion
    container: ros:${ROS_DISTRO}
```

will this work?

Reply:
I think so. Unfortunately the `ros:iron` image is the wrong one regardless because golang isn't installed.

Does it make sense to create our own image and push it to the ghcr?